spec/lib/api/2.0/config-spec.js

```diff
@@ -5,26 +5,20 @@
 
 describe('Http.Api.Config v2.0', function () {
     var configuration;
-    before('start HTTP server', function () {
-        this.timeout(10000);
-        return helper.startServer().then(function() {
-            configuration = helper.injector.get('Services.Configuration');
-            sinon.stub(configuration, 'set').returns(configuration);
-            sinon.stub(configuration, 'getAll').returns({});
-        });
-    });
 
-    afterEach('tear down mocks', function () {
-        configuration.set.reset();
-        configuration.getAll.reset();
+    helper.httpServerBefore([]);
```

Comment:
It would be better to remove the argument `[]`, since an empty array seems redundant